lib/types.ts

```diff
@@ -1,4 +1,4 @@
-import { ErrorObject } from 'serialize-error'
+import type { ErrorObject } from 'serialize-error'
 import { EventEmitter } from 'events'
```

Comment:
Could we `import type` with these, too? I see no reason why we can't...

Reply:
done via https://github.com/digidem/rpc-reflector/commit/74d369fc625b2668f84b94a9b8e8a77c6f021506